examples/webpack.config.local.js

```diff
@@ -29,16 +29,69 @@
 const resolve = require('path').resolve;
 const join = require('path').join;
 const webpack = require('webpack');
+const fs = require('fs');
+const KeplerPackage = require('../package');
+const {logStep, logError} = require('../scripts/log');
 
 const LIB_DIR = resolve(__dirname, '..');
 const SRC_DIR = resolve(LIB_DIR, './src');
 
+// For deck.gl upgrade, load deck.gl from node_modules of the root directory
 const NODE_MODULES_DIR = resolve(__dirname, '../node_modules');
 
-const KeplerPackage = require('../package');
+// For debugging deck.gl, load deck.gl from external deck.gl directory
+const EXTERNAL_DECK_SRC = resolve(__dirname, '../../deck.gl');
```

Comment:
do we need to document this in the developer guides?
also, do you think it would be better if we pass this as a paramenter when we call 

```
yarn start-local-deck-src
```

Reply:
it's in the developer.md